UniExtract.au3

```diff
@@ -3533,9 +3548,9 @@ Func HasPlugin($sPlugin, $returnFail = False)
 	_GUICtrlCreatePic($sLogoFile, 8, 20, 49, 49)
 	GUISetState(@SW_SHOW)
 
-	While 1
-		$nMsg = GUIGetMsg()
-		Switch $nMsg
```

Comment:
This was changed to `While True` for readability.
`$nMsg` was removed as it used once.
(There are multiple places where I changed these)